src/relay/ir/pretty_printer.cc

```diff
@@ -778,8 +778,13 @@ class PrettyPrinter :
     }
     Doc separator;
     separator << "," << PrintNewLine();
-    doc << Brace(PrintSep(constructor_docs, separator) << ",");
-
+    Doc adt_body;
+    adt_body << PrintSep(constructor_docs, separator);
+    // add trailing comma if there are any constructors
```

Comment:
why trailing comma?

Reply:
idk. it's not necessary. just a matter of style.